ui/page/privacy/setup_mixer_accounts_page.go

```diff
@@ -0,0 +1,323 @@
+															return layout.Inset{Bottom: values.MarginPadding12}.Layout(gtx, func(gtx C) D {
+																return ic.Layout(gtx, values.MarginPadding8)
+															})
+														}),
+														layout.Rigid(func(gtx C) D {
+															txt3 := `<span style="text-color: grayText2">
+													<b>Unmixed </b> account will be the change handling account.
+												</span>`
+
+															return layout.Inset{
+																Left: values.MarginPadding8,
+															}.Layout(gtx, renderers.RenderHTML(txt3, pg.Theme).Layout)
+														}),
+													)
+												})
+											}),
+										)
+									})
+								}),
+							)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return pg.autoSetupClickable.Layout(gtx, pg.autoSetupLayout)
+						}),
+						layout.Rigid(func(gtx C) D {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return pg.manualSetupClickable.Layout(gtx, pg.manualSetupLayout)
+						}),
+					)
+				})
+			},
+		}
+		return page.Layout(gtx)
+	}
+
+	return components.UniformPadding(gtx, body)
+}
+
+func (pg *SetupMixerAccountsPage) autoSetupLayout(gtx C) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								return pg.autoSetupIcon.Layout(gtx, values.MarginPadding20)
+							}),
+							layout.Rigid(func(gtx C) D {
+								autoSetupText := pg.Theme.H6("Auto setup")
+								txt := pg.Theme.Body2("Create and set up the needed accounts for you.")
+								return layout.Inset{
+									Left: values.MarginPadding16,
+								}.Layout(gtx, func(gtx C) D {
+									return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+										layout.Rigid(autoSetupText.Layout),
+										layout.Rigid(txt.Layout),
+									)
+								})
+							}),
+						)
+					}),
+					layout.Rigid(func(gtx C) D {
+						return layout.Flex{}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								return layout.Inset{
+									Right: values.MarginPadding4,
+									Top:   values.MarginPadding10,
+								}.Layout(gtx, func(gtx C) D {
+									return pg.nextIcon.Layout(gtx, values.MarginPadding20)
+								})
+							}),
+						)
+					}),
+				)
+			}),
+		)
+	})
+}
+
+func (pg *SetupMixerAccountsPage) manualSetupLayout(gtx C) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
```

Comment:
```suggestion
		return layout.Flex{Axis: layout.Vertical, Spacing: layout.SpaceBetween}.Layout(gtx,
```

Reply:
this you try this change out in your system? it breaks the UI
![Screenshot from 2022-03-21 15-35-13](https://user-images.githubusercontent.com/25265396/159283992-8a5454b7-100e-4fd6-bb29-be6bddfcfe47.png)

Reply:
will send a temp commit to you.

Reply:
updated. Just remove the vertical layout.